doc/source/disp-script.rst

```diff
@@ -20,7 +20,6 @@ Command Line Options
 --------------------
 
 .. argparse::
-   :filename: ../../scripts/dispersion.py
+   :module: euphonic.cli.dispersion
```

Comment:
All instances of `dispersion.py` should be changed to `euphonic-dispersion` in this file

Reply:
done!